scripts/ci/run-ci-tests.sh

```diff
@@ -301,7 +301,7 @@ if capsh --supports=cap_checkpoint_restore && unshare -c /bin/true; then
 	# under those conditions. Note that the "... && true" part is necessary; we need at least one statement after the tests so that bash can reap zombies in the user namespace,
 	# otherwise it will exec the last statement and get replaced and nobody will be left to reap our zombies.
 	sudo --user=#65534 --group=#65534 unshare -Ucfpm --mount-proc -- bash -c "./test/zdtm.py run -t zdtm/static/maps00 -f h --rootless && true"
```

Comment:
This test currently fails with the following error:
```
=[log]=> dump/zdtm/static/maps00/66/1/dump.log
------------------------ grep Error ------------------------
b'(00.000000) CRIU run id = 0xf000022600000043'
b'(00.000061) Version: 3.18 (gitid 0)'
b'(00.000070) Running on f4a8025d70c9 Linux 6.6.2-101.fc38.x86_64 #1 SMP PREEMPT_DYNAMIC Wed Nov 22 21:31:13 UTC 2023 x86_64'
b'(00.000073) Warn  (criu/kerndat.c:1153): $XDG_RUNTIME_DIR not set. Cannot find location for kerndat file'
b'(00.007422) cpu: fpu: AVX-512 ZMM_Hi256                xstate_offsets   1664 / 1408   xstate_sizes   1024 / 1024'
b'(00.007513) Effective capability 12 missing'
b'(00.008004) Found mmap_min_addr 0x10000'
b'(00.008018) files stat: fs/nr_open 1073741816'
b'(00.008022) Warn  (criu/kerndat.c:1153): $XDG_RUNTIME_DIR not set. Cannot find location for kerndat file'
b'(00.011392) Handling VMA with the following smaps entry: 7fb21b401000-7fb21b404000 rw-p 00000000 00:00 0'
b'(00.011425) Handling VMA with the following smaps entry: 7fb21b404000-7fb21b42a000 r--p 00000000 00:22 88823                      /usr/lib64/libc.so.6'
b'(00.011434) Failed to open map_files/7fb21b404000-7fb21b42a000, try to go via [/usr/lib64/libc.so.6] path'
b"(00.011456) mnt: Path `/usr/lib64/libc.so.6' resolved to `./' mountpoint"
b'(00.011458) Error (criu/proc_parse.c:379): Failed to resolve mapping 7fb21b404000 filename'
b"(00.011561) Error (criu/proc_parse.c:693): Can't open 66's mapfile link 7fb21b404000: Operation not permitted"
b'(00.011570) Error (criu/cr-dump.c:1563): Collect mappings (pid: 66) failed with -1'
b'(00.011629) net: Unlock network'
b'(00.011634) Unfreezing tasks into 1'
b'(00.011637) \tUnseizing 66 into 1'
b'(00.011655) Error (criu/cr-dump.c:2098): Dumping FAILED.'
------------------------ ERROR OVER ------------------------
```

Would it make sense to move `[ ! -e /run/.containerenv ]` in the following if statement to skip the test when running in a container?
```
# Rootless tests
# Check if cap_checkpoint_restore is supported and also if unshare -c is supported.
if capsh --supports=cap_checkpoint_restore && unshare -c /bin/true; then
```

Reply:
Yes, that is what I want to do next.